ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -289,16 +295,22 @@ extension PopupMenuController: UITableViewDataSource {
         let row = indexPath.row
         let item = sections[section].items[row]
 
-        let cell = tableView.dequeueReusableCell(withIdentifier: PopupMenuItemCell.identifier) as! PopupMenuItemCell
+        let cellClass = item.cellClass
+        let identifier = String(describing: cellClass)
+        self.tableView.register(cellClass, forCellReuseIdentifier: identifier)
```

Comment:
> self.tableView.register(cellClass, forCellReuseIdentifier: identifier) [](start = 7, length = 71)

is there a reason to register cells in this method that gets called every time we configure a cell? Why not replace the registration code we have at initTableView?

Reply:
**1. Why register in this method?**
Because this PR is to support custom cell types. 
Objective-C Class can't inherit from Swift Classes, thus we send a cellClass to dynamically register the cell.
I also thought about register the cells in addItems() and addSection(), but it will lead to more changes.
**2. Why not replace the code in initTableView?**
Just because this PR doesn't affect the headerView. Although the content has decoupled with PopupMenuItemCell already but the header still need to use the PopupMenuItemCell.
`private let headerView: PopupMenuItemCell = {
        let view = PopupMenuItemCell(frame: .zero)
        view.isHeader = true
        view.isHidden = true
        return view
    }()`